frontend/styles/var/mixins.scss

```diff
@@ -453,7 +453,7 @@ $max-width: 2560px;
   border: 1px solid $ui-fleet-black-10;
   border-radius: $pad-xsmall;
   padding: $pad-xxlarge;
-  gap: $pad-small;
+  gap: $pad-large;
```

Comment:
This looks a bit much here, wdyt?

<img width="957" height="386" alt="Screenshot 2025-09-29 at 4 28 38 PM" src="https://github.com/user-attachments/assets/39d6ab26-cfc1-4fb6-b3c8-ae89b9a33c2d" />